repose-aggregator/src/docs/asciidoc/filters/rate-limiting.adoc

```diff
@@ -1,52 +1,339 @@
-=== Shows how to ???
-This configuration will ???.
+=== Querying Limits Example
+In this example, we have configured this filter to allow users to query their limits.
+Provided below are sample requests to give you a better idea of what that looks like.
+
+.Configuration
+[source, xml]
+----
+<rate-limiting datastore="distributed/hash-ring" use-capture-groups="false" xmlns="http://docs.openrepose.org/repose/rate-limiting/v1.0">
+    <request-endpoint <!--1-->
+        uri-regex="/limits" <!--2-->
+        include-absolute-limits="false"/> <!--3-->
+
+    <limit-group id="limited" groups="BETA_Group IP_Standard" default="false">
+        <limit uri="*" uri-regex="/something/(.*)" http-methods="PUT" unit="MINUTE" value="10" />
+        <limit uri="*" uri-regex="/something/(.*)" http-methods="GET" unit="MINUTE" value="10" />
+    </limit-group>
+
+    <limit-group id="limited-all" groups="My_Group" default="true">
+        <limit uri="*" uri-regex="/something/(.*)" http-methods="ALL" unit="HOUR" value="10" />
+    </limit-group>
+</rate-limiting>
+----
+<1> By including this element, the limit query feature is enabled.
+<2> The `uri-regex` attribute specifies the URI at which queries will be handled.
+All requests that match this URI regular expression will be handled as limit query requests.
+<3> The `include-absolute-limits` attribute specifies whether or not <<Absolute Limits>> should be queried.
+
+.Limit Query Request (without group)
+----
+curl http://localhost:8020/limits -H "x-pp-user: 123456" -H "accept: application/xml"
+----
+
+.Limit Query Response (without group)
+[source, xml]
+----
+<limits xmlns="http://docs.openstack.org/common/api/v1.0">
+    <rates>
+        <rate regex="/something/(.*)" uri="*">
+           <limit next-available="2012-06-22T14:39:33.832Z" unit="HOUR" remaining="10" value="10" verb="ALL"/>
+        </rate>
+    </rates>
+</limits>
+----
+
+.Limit Query Request (with group)
+----
+curl http://localhost:8020/limits -H "x-pp-user: 123456"  -H "x-pp-groups: IP_Standard" -H "accept: application/xml"
+----
+
+[source, xml]
+.Limit Query Response (with group)
+----
+<limits xmlns="http://docs.openstack.org/common/api/v1.0">
+    <rates>
+        <rate regex="/something/(.*)" uri="*">
+            <limit next-available="2012-06-22T15:38:17.956Z" unit="MINUTE" remaining="10" value="10" verb="PUT"/>
+            <limit next-available="2012-06-22T15:38:17.956Z" unit="MINUTE" remaining="10" value="10" verb="GET"/>
+        </rate>
+    </rates>
+</limits>
+----
+
+=== Full Feature Utilization Example
+This configuration provides per-user limits, global limits, and an endpoint to query limit data.
 
 [source,xml]
-.???.cfg.xml
+.rate-limiting.cfg.xml
 ----
-<??? xmlns="http://docs.openrepose.org/repose/???/v1.0">
-    <???> <!--1-->
-    </???>
-</???>
+<rate-limiting xmlns="http://docs.openrepose.org/repose/rate-limiting/v1.0"
+    overLimit-429-responseCode="true" <!--1-->
+    datastore="distributed/hash-ring" <!--2-->
```

Comment:
It totally is:
https://github.com/rackerlabs/repose/blob/master/repose-aggregator/components/filters/rate-limiting-filter/src/main/java/org/openrepose/filters/ratelimiting/RateLimitingFilter.java#L163-L190